salt/elasticfleet/soc_elasticfleet.yaml

```diff
@@ -1,6 +1,6 @@
 elasticfleet:
   enabled:
-    description: You can enable or disable Elastic Fleet.
+    description: Enables or disables the Elastic Fleet process. This process is critical for ensuring HIDS events are made available in SOC.
```

Comment:
Enables or disables the Elastic Fleet process. This process is critical for managing Elastic Agents.